appveyor.yml

```diff
@@ -30,25 +30,27 @@ before_build:
   - 7z e xcore-win.zip -oxcore-win
   # run the tests here (instead of AppVeyor's "test" phase, which runs *after* the "build" phase)
   - xcore-win\console . schemas\dc-library.xsd
-  # now install the HTML files and tools prior to building the HTML
-  - git clone --depth 1 git@github.com:openlawlibrary/dc-law-static.git ..\dc-law-static
-  - git clone --depth 1 --branch %appveyor_repo_branch% git@github.com:dccouncil/dc-law-html.git ..\dc-law-html
+  # now install the tools and codified files prior to codifying
   - git clone --depth 1 git@github.com:openlawlibrary/dc-law-tools.git ..\dc-law-tools
+  - git clone --depth 1 git@github.com:dccouncil/dc-law-xml-codified.git ..\dc-law-xml-codified
+  # use or create a branch in dc-law-xml-codified with the same name
+  - git -C ..\dc-law-xml-codified fetch origin %appveyor_repo_branch% &&
+    git -C ..\dc-law-xml-codified checkout %appveyor_repo_branch% ||
+    git -C ..\dc-law-xml-codified checkout -b %appveyor_repo_branch%
```

Comment:
I'm going to provisionally accept this. But I'm skeptical it will work. As branches are created, merged and deleted in dc-law-xml, they will be forever accumulating in dc-law-xml-codified.